lib/rules/no-multiple-empty-lines.js

```diff
@@ -34,6 +46,12 @@ module.exports = function(context) {
                 trimmedLines = lines.map(function(str) {
                     return str.trim();
                 });
+
+            // add the notEmpty lines in there with a placeholder
+            notEmpty.forEach(function(x, i) {
```

Comment:
Do you think getting the node all the time would end up being expensive against what we have now? 
Tho it looks simpler.